dev/dependencyList

```diff
@@ -35,6 +35,7 @@ commons-logging/1.1.3//commons-logging-1.1.3.jar
 derby/10.14.2.0//derby-10.14.2.0.jar
 error_prone_annotations/2.20.0//error_prone_annotations-2.20.0.jar
 failsafe/2.4.4//failsafe-2.4.4.jar
```

Comment:
trino 363 still depends on failsafe 2.x, will upgrade trino client to remove this dep soon